pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -271,6 +275,13 @@ type attachDetachController struct {
 	csiNodeLister storagelisters.CSINodeLister
 	csiNodeSynced kcache.InformerSynced
 
+	// csiDriverLister is the shared CSIDriver lister used to fetch and store
+	// CSIDriver objects from the API server. It is shared with other controllers
+	// and therefore the CSIDriver objects in its store should be treated as immutable.
+	csiDriverLister   storagelisters.CSIDriverLister
+	csiDriverInformer storageinformers.CSIDriverInformer
```

Comment:
Why do we need the informer in the object and interface as well. Will the `Lister` suffice?

Reply:
You are right.  I get rid of informer now.